### DISCUSSION TITLE
ETA for release of firebase-monaco?

### DISCUSSION THREAD

**Discussion by Blakeinstein** [0xTheProDev/Operational-Transformation#72] [Q&A] [answered]
@hackerrank/firepad-x is frozen at monaco 0.18.x as a peer dependency. I wish to upgrade to monaco 0.28.1 but it looks like a breaking change exists.

**Comment by 0xTheProDev** [comment, marked as answer]
Hi @Blakeinstein, great to see you here.

As of this moment I don't have a tentative date for release. But I can assure you I am dedicated to bring this as soon as possible. I want to have the documentation and support up and running and the libraries being a bit more diverse in nature of APIs. But rest assured, the wait will make it even more great.

You can track [milestones](https://github.com/Progyan1997/Operational-Transformation/milestones) or [projects](https://github.com/Progyan1997/Operational-Transformation/projects) if you like to. And thanks again for showcasing your interest and that gives enough to get going.

Regards,
Progyan

PS: Can you kindly go through [Contributing Guidelines](https://github.com/Progyan1997/Operational-Transformation/blob/main/.github/CONTRIBUTING.md)?
tldr; Since we are using Issues as our tracking, we encourage people to engage in conversations in [Discussion](https://github.com/Progyan1997/Operational-Transformation/discussions) section.

**Reply by Blakeinstein** [reply]
While this looks great, is it possible for me to directly pull this from github?

**Reply by 0xTheProDev** [reply]
You can fork and build artifacts and then use them into your projects.
